.github/workflows/docker.yml

```diff
@@ -16,19 +16,25 @@ env:
 
 jobs:
   build-master-docker-ubuntu:
+    # Build images for different ubuntu versions on different platforms
+
     if: (github.event_name == 'schedule' && github.repository == 'dealii/dealii') || github.event_name != 'schedule'
 
     name: build master docker ${{ matrix.ubuntu_version }} ${{ matrix.platform }}
-    runs-on: ubuntu-24.04
+    runs-on: ${{ matrix.os }}
 
     strategy:
       fail-fast: false
       matrix:
         ubuntu_version: [jammy]
-        platform: [linux/amd64]
-        # Use only 2 jobs to build, as more jobs
-        # would starve the github machine for memory.
+        platform: [amd64, arm64]
         include:
+          - platform: amd64
+            os: ubuntu-24.04
+          - platform: arm64
+            os: ubuntu-24.04-arm64
```

Comment:
I think the correct name is `ubuntu-24.04-arm`, see: https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories

This is why you dont get a runner at the moment.

Reply:
I updated the PR. Now it seems to work. Let's see how long building each image takes!
https://github.com/marcfehling/dealii/actions/runs/13607973735